core/src/main/scala/scala/scalanative/loop/Timer.scala

```diff
@@ -1,13 +1,14 @@
 package scala.scalanative.loop
 
-import scala.scalanative.libc.stdlib
 import scala.concurrent.{Future, Promise}
 import scala.concurrent.duration._
 import LibUV._, LibUVConstants._
-import scala.scalanative.unsafe.Ptr
+import scala.scalanative.unsafe.{Ptr, sizeOf}
+import scala.scalanative.runtime.BlobArray
 import internals.HandleUtils
 
-@inline final class Timer private (private val ptr: Ptr[Byte]) extends AnyVal {
+@inline final class Timer private (private val data: BlobArray) extends AnyVal {
```

Comment:
Question: 
Should we have a registry for active timers? It makes sense for the repeated timers running for the long time in the background. If user does not keep any handle to them, then they can be GCed at some point. How often due users use them in such way? 
For safety reasons it might be best to keep them as long as the libUV might trigger the callback. Unfortunetlly we need to bridge 2 memory spaces - the SN space and libUV space.

Reply:
Perhaps @lolgab would know the answer?